source/crud/query/query-filter.txt

```diff
@@ -0,0 +1,633 @@
+these methods have alternative syntax that you can use in place of the method, as shown
+in the following example:
+
+.. code-block:: csharp
+
+   var filter = Builders<Guitar>.Filter.Eq(g => g.Make, "Fender");
+   var results = guitarCollection.Find(filter).ToList();
+
+   // Alternative syntax
+   var results = guitarCollection.Find(g => g.Make == "Fender").ToList();;
+
+The following table lists the {+driver-short+} methods for comparison operations
+and the equivalent {+mdb-server+} operators:
+
+.. list-table::
+   :widths: 20 20 40 20
+   :header-rows: 1
+
+   * - {+driver-short+} Method
+     - Alternative Syntax
+     - Description
+     - {+mdb-server+} Operator
+
+   * - ``Eq()``
+     - ``==``
+     - Matches documents where the value of the specified field is equal to the query value.
+     - :manual:`$eq </reference/operator/query/eq/>`
+
+   * - ``Gt()``
+     - ``>``
+     - Matches documents where the value of the specified field is greater than the query value.
+     - :manual:`$gt </reference/operator/query/gt/>`
+
+   * - ``Gte()``
+     - ``>=``
+     - Matches documents where any element in the specified array field is greater than or
+       equal to the query value.
+     - :manual:`$gte </reference/operator/query/gte/>`
+
+   * - ``In()``
+     - N/A
+     - Matches documents where any element in the specified array field matches any value in
+       the query array.
+     - :manual:`$in </reference/operator/query/in/>`
+
+   * - ``Lt()``
+     - ``<``
+     - Matches documents where any element in the specified array field is less than the
+       query value.
+     - :manual:`$lt </reference/operator/query/lt/>`
+
+   * - ``Lte()``
+     - ``<=``
+     - Matches documents where any element in the specified array field is less than or equal
+       to the query value.
+     - :manual:`$lte </reference/operator/query/lte/>`
+
+   * - ``Ne()``
+     - ``!=``
+     - Matches documents where any element in the specified array field is not equal to
+       the query value.
+     - :manual:`$ne </reference/operator/query/ne/>`
+
+   * - ``Nin()``
+     - N/A
+     - Matches documents where one of the following is true:
+
+       - None of the elements in the specified array field matches any of the values in the
+         query array.
+       - The specified field doesn't exist.
+     - :manual:`$nin </reference/operator/query/nin/>`
+
+   * - ``StringIn()``
+     - N/A
+     - Matches documents where the string value of the specified field matches any string
+       value in the query array.
+     - :manual:`$in </reference/operator/query/in/>`
+
+   * - ``StringNin()``
+     - N/A
+     - Matches documents where the string value of the specified field doesn't match any
+       of the string values in the query array.
+     - :manual:`$in </reference/operator/query/in/>`
+
+The following example specifies a query filter as a parameter to the ``Find()``
```

Comment:
Technically it specifies a predicate "lambda expression", which is translated by the driver into a "query filter".

But you could use a loose definition of "query filter" to include lambda expressions if you want...